src/data/actions/assignmentTypes.js

```diff
@@ -4,12 +4,17 @@ import {
   ERROR_FETCHING_ASSIGNMENT_TYPES,
   GOT_ARE_GRADES_FROZEN,
 } from '../constants/actionTypes/assignmentTypes';
+import GOT_BULK_MANAGEMENT_CONFIG from '../constants/actionTypes/config';
 import LmsApiService from '../services/LmsApiService';
 
 const startedFetchingAssignmentTypes = () => ({ type: STARTED_FETCHING_ASSIGNMENT_TYPES });
 const errorFetchingAssignmentTypes = () => ({ type: ERROR_FETCHING_ASSIGNMENT_TYPES });
 const gotAssignmentTypes = assignmentTypes => ({ type: GOT_ASSIGNMENT_TYPES, assignmentTypes });
 const gotGradesFrozen = areGradesFrozen => ({ type: GOT_ARE_GRADES_FROZEN, areGradesFrozen });
+const gotBulkManagementConfig = bulkManagementEnabled => ({
```

Comment:
If I was following the pattern super formally, I'd put this in a new `actions` file, like I've done with the action type constants. I've found, however, that grouping actions according by how their dispatched helps to avoid cyclic dependency issues, even though it's a bit less symmetrical/diverges a bit from the pattern of "all the things that correspond to this slice of state go in a file named according to the slice of state."